### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by rebornix**
first one.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodejs-shopping-cart",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "author": "George Tsopouridis",
```
Comment (RMacfarlane) [inline package.json]: asdf

**Comment by RMacfarlane** [comment]
test

**Comment by RMacfarlane** [comment]
test

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodejs-shopping-cart",
```
Comment (RMacfarlane) [inline package.json]: review comment 1

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
-  "version": "1.0.0",
+  "version": "1.0.1",
   "author": "George Tsopouridis",
   "email": "gtsopour@gmail.com",
```
Comment (RMacfarlane) [inline package.json]: review comment 2

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodejs-shopping-cart",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (RMacfarlane) [inline package.json]: test

Comment (RMacfarlane) [inline package.json]: test

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (RMacfarlane) [inline package.json]: bleep bloop

Comment (RMacfarlane) [inline package.json]: asdf

Comment (RMacfarlane) [inline package.json]: asdf

**Comment by RMacfarlane** [comment]
blah

File: package.json (hunk 2 above)
Comment (RMacfarlane) [inline package.json]: single comment in thread

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: comment 1

Comment (RMacfarlane) [reply package.json]: comment 2

**Comment by RMacfarlane** [comment]
asdf

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
...
   "name": "nodejs-shopping-cart",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "author": "George Tsopouridis",
   "email": "gtsopour@gmail.com",
   "private": false,
```
Comment (RMacfarlane) [inline package.json]: hello

Comment (RMacfarlane) [inline package.json]: asdf

**Comment by RMacfarlane** [comment]
blah

**Comment by RMacfarlane** [comment]
test

File: package.json (hunk 6 above)
Comment (Tyriar) [reply package.json]: test

File: package.json (hunk 4 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodejs-shopping-cart",
-  "version": "1.0.0",
```
Comment (RMacfarlane) [inline package.json]:
```suggestion
  "foo": "blah",

```

Comment (RMacfarlane) [inline package.json]:
```suggestion
  "foo": "blah",

```

File: package.json (hunk 4 above)
Comment (RMacfarlane) [inline package.json]:
```suggestion
  "foo": "blah",

```

Comment (RMacfarlane) [inline package.json]:
```suggestion
  "foo": "blah",

```

File: package.json (hunk 6 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 4 above)
Comment (queerviolet) [inline package.json]: lll

Comment (queerviolet) [reply package.json]: A reply

File: package.json (hunk 3 above)
Comment (queerviolet) [inline package.json]: LG

**Comment by rebornix** [comment]
leave comment

File: package.json (hunk 6 above)
Comment (RMacfarlane) [inline package.json]: hello

Comment (RMacfarlane) [reply package.json]: test

**Pull request merged**